### ISSUE TITLE
Update for 2021.1.1

### ISSUE THREAD

**Issue by rfrerebe-stx** [StyleCop/StyleCop.ReSharper#43]
Hello,

Could you please deploy a version for 2021.1.1 ?
Using Resharper 2021.1.1, I can't use this plugin in version 2021.1.0

I hope #42 helps

Thanks for your work

**Comment by matkoch** [comment]
Is this based on actually trying to use the plugin? What is the error message?

**Comment by IvanLieckens** [comment]
Not sure if it's related but: The options/settings blade for the Stylecop plugin no longer works (it shows up in the tree but when selecting it stays displaying the previously selected blade)

**Comment by rfrerebe-stx** [comment]
I updated my Resharper setup yesterday to 2021.1.1 and I am unable to active StyleCop for Resharper in the extension menu. I was on an older version of Resharper for long time before.

I am using this feed : https://resharper-plugins.jetbrains.com/api/v2/
I had 46 plugins and StyleCop was not among them.

![image](https://user-images.githubusercontent.com/77391972/114524638-adf31e80-9c45-11eb-896a-1f752c3b5fe5.png)

Yesterday, I assumed that it was because the plugin was not matching the Resharper version.
Today, I changed the feed to https://resharper-plugins.jetbrains.com/api/v2/curated-feeds/Wave_v211.0/ and I was able to get the plugin

![image](https://user-images.githubusercontent.com/77391972/114528661-80a86f80-9c49-11eb-9ab2-0ad39d970496.png)
